### PR TITLE
Hotspot search

### PR DESCRIPTION
**Pull request by ambuznego**

### CODE REVIEW

File: assets/js/Router.jsx [hunk 1]
```diff
@@ -336,7 +336,7 @@ const Router = (props) => {
                                 <Profile user={user} {...props} />
                               )}
                             />
-                            {false && (
+                            {true && (
```
Comment (vicmgs) [inline assets/js/Router.jsx:339]: swap back lool

Comment (ambuznego) [reply assets/js/Router.jsx:339]: i cry

**Review by vicmgs — APPROVED** [review]
niiice

File: assets/js/Router.jsx (hunk 1 above)
Comment (vicmgs) [reply assets/js/Router.jsx]: i literally deployed true to staging too sigh

**Pull request merged**